pages/dkp/dispatch/1.1/release-notes/open-source-attribution/index.md

```diff
@@ -146,6 +146,6 @@ Software that are utilized by D2IQ Dispatch.
 
 | Software       | Software Source                                              | License                                                                                      |
 | -------------- | ------------------------------------------------------------ | -------------------------------------------------------------------------------------------- |
-| golang-lru     | https://packages.d2iq.com/ksphere/golang-lru-0.5.4.tar.gz    | [Mozilla Public License 2.0](https://github.com/hashicorp/golang-lru/blob/v0.5.4/LICENSE)    |
-| errwrap        | https://packages.d2iq.com/ksphere/errwrap-1.0.0.tar.gz       | [Mozilla Public License 2.0](https://github.com/hashicorp/errwrap/blob/v1.0.0/LICENSE)       |
-| go-multierror  | https://packages.d2iq.com/ksphere/go-multierror-1.1.0.tar.gz | [Mozilla Public License 2.0](https://github.com/hashicorp/go-multierror/blob/v1.1.0/LICENSE) |
+| golang-lru     | https://packages.d2iq.com/dkp/golang-lru-0.5.4.tar.gz    | [Mozilla Public License 2.0](https://github.com/hashicorp/golang-lru/blob/v0.5.4/LICENSE)    |
+| errwrap        | https://packages.d2iq.com/dkp/errwrap-1.0.0.tar.gz       | [Mozilla Public License 2.0](https://github.com/hashicorp/errwrap/blob/v1.0.0/LICENSE)       |
+| go-multierror  | https://packages.d2iq.com/dkp/go-multierror-1.1.0.tar.gz | [Mozilla Public License 2.0](https://github.com/hashicorp/go-multierror/blob/v1.1.0/LICENSE) |
```

Comment:
Im not sure what these packages are, but they are not available in `https://packages.d2iq.com/dkp/golang-lru-0.5.4.tar.g`

Reply:
yes, got overzealous. Changed back.